app/Models/Beatmap.php

```diff
@@ -298,9 +298,10 @@ public function getOwners(): Collection
             fn ($beatmapOwner) => $beatmapOwner->user ?? new DeletedUser(['user_id' => $beatmapOwner->user_id])
         );
 
+        // Fallback if unpopulated
         // TODO: remove when everything writes to beatmap_owners.
```

Comment:
not sure the pr title is relevant anymore. also this todo should be adjusted/removed depending on how empty owners should be handled